README.md

```diff
@@ -74,6 +74,7 @@ Cling(hostname=None,
     - Personalities supported:
         - `generic`       Generic personality (does not run init or exit commands)
         - `ios`           Cisco IOS
+        - `iosxe`         Cisco IOS XE
```

Comment:
Is this based on master? On master, the iosxe is added under the iosxr. Please check and fix.